src/containers/navigation/menu/index.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from 'react';
+
+import Image, { StaticImageData } from 'next/image';
+
+import cn from 'lib/classnames';
+
+import { AnimatePresence, motion } from 'framer-motion';
+
+import BlogContent from 'containers/blog/content';
+import Helper from 'containers/guide/helper';
+import About from 'containers/navigation/menu/about';
+
+import { Dialog, DialogContent, DialogClose, DialogTrigger } from 'components/dialog';
+import Icon from 'components/icon';
+import { Media } from 'components/media-query';
+
+import GMA_PNG from 'images/gma.png';
+
+import ABERYSTWYTH_SVG from 'svgs/partners/aberystwyth.svg?sprite';
+import NATURE_CONSERVANCY_SVG from 'svgs/partners/nature.svg?sprite';
+import SOLO_SVG from 'svgs/partners/solo.svg?sprite';
+import WETLANDS_SVG from 'svgs/partners/wetlands.svg?sprite';
+import MENU_SVG from 'svgs/tools-bar/menu.svg?sprite';
+
+const Menu = () => {
+  const [section, setSection] = useState('main');
+
+  return (
+    <Dialog>
+      <Helper
+        className={{
+          button: '-bottom-10 -right-1.5',
+          tooltip: 'w-fit-content',
+        }}
+        tooltipPosition={{ top: -10, left: -60 }}
+        message="main menu"
+      >
+        <DialogTrigger asChild>
+          <button
+            data-testid="menu-button"
+            type="button"
+            onClick={() => setSection('main')}
+            className="flex items-center space-x-2"
+          >
+            <Icon icon={MENU_SVG} className="h-6 w-6 " description="Menu" />
+            <span className="font-sans text-xs text-white">Menu</span>
+          </button>
+        </DialogTrigger>
+      </Helper>
+
+      <DialogContent
+        data-testid="menu-content"
+        className={cn({
```

Comment:
This code avoid weird animation and set all true code in same line:

```suggestion
        className={cn({
          'scroll-y rounded-3xl px-10 font-sans md:top-[10%] md:max-w-xl': true,
          'h-fit py-0': section === 'main',
          'h-[96%] pb-0 md:h-[90vh] md:py-0': section === 'news' || section === 'about',
        })}
```